book/getting-started/download-development-project.md

```diff
@@ -43,9 +43,13 @@ __Note__: before testing make sure that the project is working for you out of th
 If you are adding files or assets in your development pod - run `pod update YourPlugin --no-repo-update` to update your changes in the project.
 
 #### External developers with no access to Applicaster private repositories (EXPERIMENTAL)
-1. Update the `Podfile` to include your `podspec` local path.
-2. In the `Podfile` comment out `source 'git@github.com:applicaster/CocoaPods-Private.git'`
-3. Run `pod update YourPlugin --no-repo-update`.
+1. Ensure that the plugin you are working on has been installed on Zapp
+    1. Go to the version you are working on -> Plugin configuration, find your plugin and add to build.
+    2. If you added the plugin in the step above, run the build again following the steps above. 
+2. Download the project, and edit the podfile:
+    1. Comment out your pod inserted by Zapptool and add another line pointing to your local version
```

Comment:
external develop doesn't know what it is `zapptool`

Reply:
Fair point, amended to include where to look.